apps/notification/models.py

```diff
@@ -73,3 +75,31 @@ class Meta:
     @staticmethod
     def get_for(user):
         return Notification.objects.filter(receiver=user).distinct()
+
+
+class Assignment(models.Model):
+    """
+    Assignment Model
+    """
+    timestamp = models.DateTimeField(
+        default=timezone.now,
+    )
+    created_for = models.ForeignKey(User, on_delete=models.CASCADE)
+    project = models.ForeignKey(
+        Project,
+        on_delete=models.CASCADE,
+        blank=True,
+        null=True,
+        default=None
+    )
+    is_done = models.BooleanField(default=False)
+    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE)
+    object_id = models.PositiveIntegerField()
+    content_object = GenericForeignKey('content_type', 'object_id')
+
```

Comment:
let's also define the `created_by` field, where we save the user whose request triggered this assignment?
cc: @annshress @AdityaKhatri @samshara

Reply:
Yes, that is needed.

Reply:
Done